packages/components/src/DataList/components/DataListLayoutInternal/DataListLayoutInternal.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { BREAKPOINTS, Breakpoints } from "../../DataList.const";
+import { DataListObject } from "../../DataList.types";
+import { sortSizeProp } from "../../DataList.utils";
+import { DataListLayoutProps } from "../DataListLayout/DataListLayout";
+
+/**
+ * Determine is layout is visible by checking a media query matches for the
+ * visible sizes of the layout and there isn't a larger layout that should be rendered
+ */
+function isLayoutVisible({
```

Comment:
gotta admit, did not expect to see this function first. For consistency's sake again, probs should drop this below the component that this file name is referencing.